### DISCUSSION TITLE
What about creating vitepress seo plugin

### DISCUSSION THREAD

**Discussion by narumi147** [vuepress-theme-hope/vuepress-theme-hope#3572] [Ideas]
So far, I didn't find a usable seo plugin for vitepress. Wondering if it's possible to migrate from vuepress seo2 plugin?
Other plugins are also wonderful if support vitepress. 🙏🏻

**Comment by Mister-Hope** [comment]
I do not have time developing any vitepress related things for the next 4 month at least.

Busy with my article and finding a job.
